Dockerfile

```diff
@@ -1,49 +1,47 @@
 FROM golang:1.21 AS base
 
-	WORKDIR /mint/
-
-	# Copy certificates
-	COPY config/tls/*.crt /usr/local/share/ca-certificates/
-	RUN update-ca-certificates && \
-		apt install tzdata && \
-		rm -rf /var/lib/apt/lists/*
-	
-	# Copy go.mod and go.sum files to the workspace and download dependencies
-	COPY go.mod ./
-	COPY go.sum ./
-	RUN go mod download
+WORKDIR /mint/
+
+# Copy certificates
+COPY config/tls/*.crt /usr/local/share/ca-certificates/
+RUN update-ca-certificates && \
+	apt update && \
+	apt install -y tzdata && \
+	rm -rf /var/lib/apt/lists/*
+
+# Copy go.mod and go.sum files to the workspace and download dependencies
+COPY go.mod go.sum ./
+RUN go mod download
 
 FROM base AS dev
 
-	# Install delve, gopls, and air for debugging
-	RUN go install github.com/go-delve/delve/cmd/dlv@latest && \
-		go install golang.org/x/tools/gopls@latest && \
-		go install github.com/cosmtrek/air@4612c12f1ed7c899314b8430bc1d841ca2cb061a
+# Install delve, gopls, and air for debugging
+RUN go install github.com/go-delve/delve/cmd/dlv@latest && \
+	go install golang.org/x/tools/gopls@latest && \
+	go install github.com/cosmtrek/air@4612c12f1ed7c899314b8430bc1d841ca2cb061a
 
 FROM base AS build
 
-	COPY cmd ./cmd
-	COPY pkg ./pkg
-	RUN CGO_ENABLED=0 GOOS=linux go build -a -o bin/mint ./cmd/mint
+COPY cmd/ pkg/ ./
+RUN CGO_ENABLED=0 GOOS=linux go build -a -o bin/mint ./cmd/mint
 
 FROM scratch
 
-	WORKDIR /mint/
-
-	COPY --from=build /mint/bin/mint .
-	COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-	COPY --from=build /usr/share/zoneinfo /usr/share/zoneinfo
+WORKDIR /mint/
 
-	# Copy build args to envs for version, datetime, and timestamp
-	ARG ARG_APPLICATION_VERSION
-	ARG ARG_APPLICATION_DATETIME
-	ARG ARG_APPLICATION_TS
-	ENV APPLICATION_VERSION=${ARG_APPLICATION_VERSION}
-	ENV APPLICATION_DATETIME=${ARG_APPLICATION_DATETIME}
-	ENV APPLICATION_TS=${ARG_APPLICATION_TS}
+COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=build /usr/share/zoneinfo /usr/share/zoneinfo
+COPY --from=build /mint/bin/mint ./
```

Comment:
Does re-ordering (or adding the trailing slash to the destination) accomplish anything here? Fine with the change itself, but was curious if there was a performance benefit to doing it in this order (I suppose the binary is more likely to differ than the zoneinfo / certs, so putting it later results in better caching of the final image?)

Reply:
This was a mistake! I wanted to reorganize these layers, but ALSO pull certs and tzinfo from base instead of build. This should help caching of those layers.